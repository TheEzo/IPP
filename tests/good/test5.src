.IPPcode18
labeL $a
label -a
label _a
label *a
label %a
label &a
label aksdjf