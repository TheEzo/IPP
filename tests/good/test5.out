<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode18">
 <instruction order="1" opcode="LABEL">
  <arg1 type="label">$a</arg1>
 </instruction>
 <instruction order="2" opcode="LABEL">
  <arg1 type="label">-a</arg1>
 </instruction>
 <instruction order="3" opcode="LABEL">
  <arg1 type="label">_a</arg1>
 </instruction>
 <instruction order="4" opcode="LABEL">
  <arg1 type="label">*a</arg1>
 </instruction>
 <instruction order="5" opcode="LABEL">
  <arg1 type="label">%a</arg1>
 </instruction>
 <instruction order="6" opcode="LABEL">
  <arg1 type="label">&amp;a</arg1>
 </instruction>
 <instruction order="7" opcode="LABEL">
  <arg1 type="label">aksdjf</arg1>
 </instruction>
</program>
